core/plugins/org.polarsys.capella.core.ui.toolkit/src/org/polarsys/capella/core/ui/toolkit/dialogs/ConfirmDeleteCapellaElementDialog.java

```diff
@@ -284,12 +285,22 @@ protected Control createContents(Composite parent) {
   private void updateMessageAndButtons() {
     IStatus status = getStatus();
     if (!status.isOK()) {
```

Comment:
I think this shall occurs only on ERROR status.
(if you have a warning, does the OK shall be disabled?)